stubs/seaborn/seaborn/_core/groupby.pyi

```diff
@@ -7,7 +7,6 @@ from pandas import DataFrame
 
 # pandas._typing.AggFuncTypeFrame is partially Unknown
 _AggFuncTypeBase: TypeAlias = Callable[..., Incomplete] | str | ufunc
-# Using Hashable instead of HashableT to work around pytype issue
 _AggFuncTypeDictFrame: TypeAlias = Mapping[Hashable, _AggFuncTypeBase | list[_AggFuncTypeBase]]
```

Comment:
doesn't this comment imply that we should try using `HashableT` here? 😄

Reply:
Yes, but `HashableT` doesn't exist in any useful capacity in any of the seaborn stubs (or `_typeshed`). So I'm not sure what introducing it here would look like. I assume this is an old comment that's become obsolete due to changes to the stubs in the meantime.

Reply:
The comment was added in #10974. It looks like @Avasam originally tried to use `HashableT` from `pandas._typing` in that PR, but switched to using `Hashable` to placate some pytype issues in this commit on the PR branch ("fix pytype"): https://github.com/python/typeshed/pull/10974/commits/b25381c728292337791c74216a09eaa4a45022a7

Reply:
I'm not sure making the type alias generic would actually have any practical user-facing difference to what you have, though. So I'm okay with landing this PR as you currently have it!

Reply:
Yeah that sounds about right